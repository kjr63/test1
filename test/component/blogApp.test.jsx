import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import BlogApp from '../../public_html/jsx/component/blogApp.jsx';

test (
	'should return html-code',
	function () {
		const renderer = new ReactShallowRenderer();
		renderer.render(<BlogApp />);
		expect(renderer.getRenderOutput()).toMatchSnapshot();
	}
);
