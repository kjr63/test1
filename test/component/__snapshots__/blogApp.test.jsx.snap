// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return html-code 1`] = `
<div
  className="mainroot"
>
  <div>
    <a
      href="index.html"
    >
      Aloitussivulle
    </a>
  </div>
</div>
`;
